meta-mbl-distro/recipes-core/packagegroups/packagegroup-mbl-production.bb

```diff
@@ -58,7 +56,6 @@ PACKAGEGROUP_MBL_PRODUCTION_PKGS_append = " optee-client"
 PACKAGEGROUP_MBL_PRODUCTION_PKGS_append = " connman"
 PACKAGEGROUP_MBL_PRODUCTION_PKGS_append = " connman-client"
 PACKAGEGROUP_MBL_PRODUCTION_PKGS_append = " systemd-net-conf"
-PACKAGEGROUP_MBL_PRODUCTION_PKGS_append = " ${@bb.utils.contains('MACHINE_FEATURES', 'ecryptfs', 'ecryptfs-utils', '', d)}"
-PACKAGEGROUP_MBL_PRODUCTION_PKGS_append = " ${@bb.utils.contains('MACHINE_FEATURES', 'ecryptfs', 'keyutils', '', d)}"
+PACKAGEGROUP_MBL_PRODUCTION_PKGS_append = " ${@bb.utils.contains('MACHINE_FEATURES', 'psa-storage', 'psa-trusted-storage-linux', '', d)}"
```

Comment:
I agree we should have a separate package to include the example binary and this package should be included in the development packagegroup since it is an example and not a feasible-to-production application.

Reply:
The logical implication of not adding this line it not to accept this PR into the warrior-dev. I'm OK with that.